src/analysis/processing/qgsalgorithmdxfexport.cpp

```diff
@@ -87,7 +87,7 @@ QVariantMap QgsDxfExportAlgorithm::processAlgorithm( const QVariantMap &paramete
 
   const QgsDxfExport::SymbologyExport symbologyMode = static_cast< QgsDxfExport::SymbologyExport >( parameterAsInt( parameters, QStringLiteral( "SYMBOLOGY_MODE" ), context ) );
   const double symbologyScale = parameterAsDouble( parameters, QStringLiteral( "SYMBOLOGY_SCALE" ), context );
-  const QString encoding = QgsDxfExport::encodings().at( parameterAsInt( parameters, QStringLiteral( "ENCODING" ), context ) );
+  const QString encoding = parameterAsString( parameters, QStringLiteral( "ENCODING" ), context );
```

Comment:
I think this should be parameterAsEnumString

Reply:
Done, thanks.